bundles/org.openhab.binding.wiz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,208 @@
+				<option value="4">Party</option>
+				<option value="5">Fireplace</option>
+				<option value="6">Cozy White</option>
+				<option value="7">Forest</option>
+				<option value="8">Pastel Colors</option>
+				<option value="9">Wakeup</option>
+				<option value="10">Bed Time</option>
+				<option value="11">Warm White</option>
+				<option value="12">Daylight</option>
+				<option value="13">Cool White</option>
+				<option value="14">Night Light</option>
+				<option value="15">Focus</option>
+				<option value="16">Relax</option>
+				<option value="17">True Colors</option>
+				<option value="18">TV Time</option>
+				<option value="19">Plant Growth</option>
+				<option value="20">Spring</option>
+				<option value="21">Summer</option>
+				<option value="22">Fall</option>
+				<option value="23">Deep Dive</option>
+				<option value="24">Jungle</option>
+				<option value="25">Mojito</option>
+				<option value="26">Club</option>
+				<option value="27">Christmas</option>
+				<option value="28">Halloween</option>
+				<option value="29">Candlelight</option>
+				<option value="30">Golden White</option>
+				<option value="31">Pulse</option>
+				<option value="32">Steampunk</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="light-mode-speed">
+		<item-type>Dimmer</item-type>
+		<label>Dynamic Light Mode Speed</label>
+		<description>Speed of color/intensity changes in dynamic light modes</description>
+		<tags>
+			<tag>Lighting</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="fan-speed">
+		<item-type>Number</item-type>
+		<label>Fan Speed</label>
+		<description>Speed of the fan, in arbitrary steps</description>
+		<category>Fan</category>
+		<state min="0" max="6" pattern="%.0f"/>
+	</channel-type>
+
+	<channel-type id="fan-reverse">
+		<item-type>Switch</item-type>
+		<label>Reverse</label>
+		<state>
+			<options>
+				<option value="OFF">Forward</option>
+				<option value="ON">Reverse</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fan-mode">
+		<item-type>Number</item-type>
+		<label>Mode</label>
+		<state>
+			<options>
+				<option value="1">Normal</option>
+				<option value="2">Breeze</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="last-update">
+		<item-type>DateTime</item-type>
+		<label>Last Update</label>
+		<description>Timestamp of last status update</description>
+		<category>Time</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-group-type id="device-channels">
+		<label>Device</label>
+		<channels>
+			<channel id="signal-strength" typeId="system.signal-strength"/>
+			<channel id="last-update" typeId="last-update"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="dimmable-light">
+		<label>Light</label>
+		<channels>
+			<channel id="brightness" typeId="system.brightness"/>
+			<channel id="mode" typeId="light-mode"/>
+			<channel id="speed" typeId="light-mode-speed"/>
```

Comment:
This is quite intentional. According to openHAB guidelines, when a light supports brightness, it should _only_ have the dimmer channel, not a switch+dimmer channel. If you somehow have a non-dimmable thing hooked up to a device that normally supports a dimmable thing, you can always just link a switch item to a dimmer channel. Dimmer items can still also accept ON/OFF commands, and some UIs have a way to present that (I'm more familiar with BasicUI rather than MainUI; for BasicUI it's the `switchSupport` option on a `Slider` widget; `Colorpicker` widgets also present an "on" and "off" button of some form when trying to send an ON or OFF command to a Color item). You can also just connect a switch widget to a dimmer item.